io/plugins/eu.esdihumboldt.hale.io.csv/plugin.xml

```diff
@@ -156,7 +156,16 @@
             name="CSV file">
          <contentType
                ref="eu.esdihumboldt.hale.io.csv">
-         </contentType>
+         </contentType>        
+         <providerParameter
```

Comment:
Please add a `valueDescriptor` element that specifies what the default value is.